check_pgactivity

```diff
@@ -5551,6 +5558,18 @@ sub check_streaming_delta {
 
     @rs = @{ query_ver( $hosts[0], %queries ) };
 
+    my $index = 0;
+    if (@exclude){
+        foreach my $line (@rs){
+	    foreach my $exclude_re (@exclude){
+	        if (grep { $_ =~ $exclude_re } @{$line}){
```

Comment:
I'd prefer to explicitely test the regexp on the application_name and client_addr fields (and update the related comment at the beginning of this function).